client-app/src/components/DraftOrcaDashboard.js

```diff
@@ -101,8 +102,19 @@ const DraftOrcaDashboard = () => {
       file_path: filePath.toString(),
       search_terms: searchTerms,
       sections: sections,
-      specify_lines: specifyLines.join(","),
+      specify_lines: specifyLines
```

Comment:
Can you please simplify the code? This looks a bit complex and difficult to maintain. Please implement the following changes:

- Add a function for the formatting part
- Move the formatting logic to formatSpecifyLines()
- Call the method in specify_lines parameter as: specify_lines: formatSpecifyLines()